hooks/airbrake/airbrake.go

```diff
@@ -1,51 +1,51 @@
-package logrus_airbrake
+package airbrake
 
 import (
+	"errors"
+	"fmt"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/tobi/airbrake-go"
```

Comment:
Does this get confusing when they both use `package airbrake`? Should we import this under a different name?